test/config/ApplicationConfigSpec.scala

```diff
@@ -28,4 +28,15 @@ class ApplicationConfigSpec extends TAMCSetupSpec {
 
   }
 
+  "ggSignInUrl" must {
+    "build ggSignInUrl" when {
+      "has no continue" in {
+        ApplicationConfig.ggSignInUrl(None) shouldBe "/gg/sign-in"
+      }
+      "has continue" in {
+        ApplicationConfig.ggSignInUrl(Some("/tamc")) shouldBe "/gg/sign-in?continue=%2Ftamc"
```

Comment:
Question relation to whether this should be an option

Reply:
:+1: